### PR TITLE
Notifications Instructions

### PR DESCRIPTION
**Pull request by micahalcorn**
For reference 📖

### CODE REVIEW

File: origin-notifications/README.md [hunk 1]
```diff
@@ -5,3 +5,54 @@ Head to https://www.originprotocol.com/developers to learn more about what we're
...
+  ```
+  NOTIFICATIONS_KEY=XXXXXXXXXX-your-public-key-XXXXXXXXXX
+  NOTIFICATIONS_URL=http://localhost:3456/
+  ```
+  - Run `npm start`
+  
+In origin-discovery...
+  
+  - Run `node src/listener/listener.js --continue-file=continue --webhook=http://localhost:3456/events`
+
```
Comment (Natasha08) [inline origin-notifications/README.md]: I have at least one undefined variable
image

I think it is the `DATABASE_URL`.
- Is this the correct format `postgresql://localhost/notification`?
- Is there another variable I need?

Comment (micahalcorn) [reply origin-notifications/README.md]: Ah, I forgot one. Thanks for testing. 😁

Comment (micahalcorn) [reply origin-notifications/README.md]: @Natasha08 try now with 37dd006 🤞

**Comment by micahalcorn** [comment]
@Natasha08 have you taken some action that would trigger a notification? For example, subscribe to notifications from a seller account in browser A and then create an offer from a buyer account in browser B?

**Comment by Natasha08** [comment]
@micahalcorn Yes I have been doing the example you mentioned. The origin notifications widget shows the notification, but after getting the browser window asking for permission to show notifications, I do not see browser notifications

**Pull request merged**